src/30_Advanced/SeatMap.html

```diff
@@ -108,29 +116,26 @@ <h1>Seat map</h1>
           As we don't know the actual size of the seatmap, we are using `layout=fill` for `amp-list` and `amp-pan-zomm` to allow the seatmap to use all the available space in its container, which we set to 50% of the viewport height.
           ~-->
         <amp-pan-zoom layout="fill" id="pan-zoom">
-          <svg class="seat-map-svg" preserveAspectRatio="xMidYMin slice" viewBox="0 0 {{width}} {{height}}">
-          {{#seats}}                
-          <!--~
-          If a seat is selected, when the user selects it again, it's removed from the selectedSeats list
-          ~-->
-            <rect class="seat {{unavailable}}" [class]="selectedSeats.includes('{{id}}')? 'seat selected' : 'seat {{unavailable}}'" 
-              x="{{x}}" y="{{y}}" width="{{width}}" height="{{height}}" 
-              role="button" 
-              tabindex="0" rx="{{rx}}" ry="{{ry}}"
-              on="{{#available}} tap:AMP.setState({
-                  selectedSeats: !selectedSeats.includes('{{id}}') ? 
-                  selectedSeats.concat('{{id}}') : copyAndSplice(selectedSeats, selectedSeats.indexOf('{{id}}'), 1)
-              }) {{/available}}"></rect>  
+          <amp-selector multiple
+                on="select:AMP.setState({
+                    selectedSeats: event.selectedOptions})">
```

Comment:
nit: format the action -> add new line before the closing `}`

Reply:
done